### ISSUE TITLE
Use ballon example to ADD a class to the pre-trained (with coco) Mask_RCNN

### ISSUE THREAD

**Issue by TasosR83** [matterport/Mask_RCNN#1732]
[Balloon example ](https://engineering.matterport.com/splash-of-color-instance-segmentation-with-mask-r-cnn-and-tensorflow-7c761e238b46) explains perfect how train the Mask_RCNN creating a new class to the model that has been trained with the COCO dataset.

Suppose we need to add the balloon class to the 80 (I think) categories that already the COCO contains, how should I start ? Is it something easy to-do ?

Duplicate of the [#828](https://github.com/matterport/Mask_RCNN/issues/828) (which is still open).

Please CLOSE this issue
